### DISCUSSION TITLE
Create folder inside a drive

### DISCUSSION THREAD

**Discussion by SaketMunda** [deta/cloud-docs#221] [Help] [answered]
is that possible to create folder inside an existing deta drive through Python ?

**Comment by jnsougata** [comment, marked as answer]
yes it's possible... just add folder name with file name while uploading a file.
for example if your file name is `sample.png` and folder name is `test` then during file upload you have to put file name `test/sample.png`
if `test` folder doesn't exist it will be created automatically or if it is already there in the drive the `sample.png` will be saved inside it
note: an empty folder can not be created
